scp.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	scp "github.com/bramvdbogaerde/go-scp"
+	"github.com/bramvdbogaerde/go-scp/auth"
+	"golang.org/x/crypto/ssh"
+	"os"
+	"time"
+)
+
+func scpToWorker(source string, dest string, tarName string) error {
+	// Use SSH key authentication from the auth package
+	// we ignore the host key in this example, please change this if you use this library
+	clientConfig, err := auth.PrivateKey("ubuntu", "/home/ubuntu/.ssh/senior-design.pem", ssh.InsecureIgnoreHostKey())
```

Comment:
Do not hardcode paths! Also, is there an alternative to "InsecureIgnoreHostKey" / is it actually insecure?